### DISCUSSION TITLE
brigde

### DISCUSSION THREAD

**Discussion by BingXVNN** [FuelLabs/sway#5287] [General]
The time brigde to the fuel wallet is very fast, but the time it takes to withdraw money to the wallet is very long

**Comment by vapsvapsov** [comment]
and also theres problems with fees when u try to claim a lot of tokens, yeah
![image](https://github.com/FuelLabs/sway/assets/128076915/558e2573-8337-45b0-88b7-8cfd59b80881)

**Reply by kendu3310** [reply]
Mine is faster

**Comment by Nar035** [comment]
Its solved?
